integreat_cms/cms/templates/poicategories/poicategory_form.html

```diff
@@ -39,4 +64,6 @@ <h3 class="heading font-bold text-black">
             </div>
         </div>
     </form>
+    {% include "generic_confirmation_dialog.html" %}
+    {{ media_config_data|json_script:"media_config_data" }}
```

Comment:
```suggestion
```
Since we only have a fixed set of possible icons, we don't need the media context here